charts/opentelemetry-demo/values.yaml

```diff
@@ -375,7 +379,7 @@ components:
       - name: PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION
         value: python
       - name: OTEL_EXPORTER_OTLP_ENDPOINT
-        value: 'http://{{ include "otel-demo.name" . }}-otelcol:4317'
+        value: 'http://{{ include "otel-demo.name" . }}-otelcol:4318/v1/traces'
```

Comment:
According to the spec `OTEL_EXPORTER_OTLP_ENDPOINT` should not use the full endpoint as it will be interpreted as a base and have `/v1/metrics` and `/v1/traces` appended automatically.  What changed in the demo to require the full url?  Should we use `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`  instead?

Reply:
good catch.  Moving this to `OTLP_EXPORTER_TRACES_ENDPOINT` instead.